### PR TITLE
XPollCode should not appear in Navigation Panel #26

### PR DESCRIPTION
**Pull request by oanalavinia**
afternavigationpanel

### CODE REVIEW

File: application-xpoll-test/application-xpoll-test-tests/src/test/it/org/xwiki/xpoll/test/ui/XPollTest.java [hunk 1]
```diff
@@ -136,7 +136,7 @@ private CreatePagePage createPage(XPollHomePage xpollHomePage)
     {
         CreatePagePage createPage = xpollHomePage.createPage();
         createPage.getDocumentPicker().setTitle(pollName);
-        createPage.setTemplate("XPollCode.XPollTemplateProvider");
+        createPage.setTemplate("XPoll.XPollCode.XPollTemplateProvider");
```
Comment (mflorea) [inline application-xpoll-test/application-xpoll-test-tests/src/test/it/org/xwiki/xpoll/test/ui/XPollTest.java]: The [best practice](https://dev.xwiki.org/xwiki/bin/view/Community/ApplicationDevelopmentBestPractices#HPages) is to use the ``Code`` space. So it should be:

    XPoll.Code.XPollTemplateProvider

File: application-xpoll-ui/src/main/resources/XPoll/WebPreferences.xml [hunk 2]
```diff
@@ -1460,7 +1460,7 @@
       <registration_registered>---</registration_registered>
     </property>
     <property>
-      <rightPanels>XPollCode.RecentPollsPanel</rightPanels>
+      <rightPanels>XPoll.XPollCode.RecentPollsPanel</rightPanels>
```
Comment (mflorea) [inline application-xpoll-ui/src/main/resources/XPoll/WebPreferences.xml]: Same as above.

File: application-xpoll-ui/src/main/resources/XPollCode/RecentPollsPanel.xml [hunk 3]
```diff
@@ -20,15 +20,15 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.2" reference="XPollCode.RecentPollsPanel" locale="">
-  <web>XPollCode</web>
+<xwikidoc version="1.2" reference="XPoll.XPollCode.RecentPollsPanel" locale="">
```
Comment (mflorea) [inline application-xpoll-ui/src/main/resources/XPollCode/RecentPollsPanel.xml]: Same as above. The page should be in the ``Code`` subspace. But what's even more important is that the page reference needs to match the location of the page on the file system. The page is still in ``resources/XPollCode/``. It should be in ``/resources/XPoll/Code/``.

File: application-xpoll-ui/src/main/resources/XPollCode/XPollTemplateProvider.xml [hunk 4]
```diff
@@ -19,8 +19,8 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.3" reference="XPollCode.XPollTemplateProvider" locale="">
-  <web>XPollCode</web>
+<xwikidoc version="1.3" reference="XPoll.XPollCode.XPollTemplateProvider" locale="">
+  <web>XPoll.XPollCode</web>
```
Comment (mflorea) [inline application-xpoll-ui/src/main/resources/XPollCode/XPollTemplateProvider.xml]: Same as above. The page reference doesn't match the page location on the file system.

File: application-xpoll-ui/src/main/resources/XPollCode/RecentPollsPanel.xml (hunk 3 above)
Comment (oanalavinia) [reply application-xpoll-ui/src/main/resources/XPollCode/RecentPollsPanel.xml]: So Code should be moved from resouces/ to resouces/XPoll space?

Comment (mflorea) [reply application-xpoll-ui/src/main/resources/XPollCode/RecentPollsPanel.xml]: You just need to keep the path you have in the exported XAR. The issue is that you copied the XML from the XAR where is has the new path to the source tree where is has the old path.

**Pull request merged**